### ISSUE TITLE
Unsigned numeric types are not serialized correctly

### ISSUE THREAD

**Issue by gdude2002** [Peanuuutz/tomlkt#13]
Here's an odd one for you. I've got the following data class in my project:

```kotlin
@Serializable
@Suppress("DataClassShouldBeImmutable")
data class PKGuildConfig(
    var apiUrl: String = "https://api.pluralkit.me",
    @Suppress("MagicNumber, UnderscoresInNumericLiterals")
    var botId: ULong = 466378653216014359UL,
    var enabled: Boolean = true,
) : Data
```

`Data` is an interface needed to clobber the type system into doing what I need, but it contains nothing.

If we take this and serialize it to TOML, here's what we get:

```toml
apiUrl = "https://api.pluralkit.me"
466378653216014359
enabled = true
```

What we'd expect here would be:

```toml
apiUrl = "https://api.pluralkit.me"
botId = 466378653216014359
enabled = true
```

If you use a signed type, like `Long`, this doesn't happen. Weird.

Any chance of a fix for this?

**Comment by Peanuuutz** [comment]
Oops, seems like inline classes are not processed properly. I'll fix this tomorrow.

**Comment by Peanuuutz** [comment]
Fixed with #14.

**Comment by gdude2002** [comment]
I can confirm this fix. Thanks for your hard work!
